src/org/jitsi/impl/neomedia/rtp/translator/SimulcastController.java

```diff
@@ -274,17 +281,34 @@ public void setTargetIndex(int newTargetIdx)
                 return;
             }
 
+            RTPEncodingDesc[] sourceEncodings = sourceTrack.getRTPEncodings();
+
             int currentTL0Idx = bitstreamController.getCurrentIndex();
             if (currentTL0Idx > -1)
             {
-                currentTL0Idx = sourceTrack.getRTPEncodings()
-                    [currentTL0Idx].getBaseLayer().getIndex();
+                currentTL0Idx
```

Comment:
Temporal Layer 0; simulcast doesn't have to know anything about higher temporal layers.